config/BloodMagic/meteors/MASTER1_GemMaster.json

```diff
@@ -0,0 +1,61 @@
+{
+  "ores": [
+	"oreEndstoneDiamond",
+    "5",
+	"oreEndstoneEmerald",
+    "5",
+	"oreEndstoneRuby",
+    "5",
+	"oreEndstoneSapphire",
+    "5",
+	"oreEndstoneGreenSapphire",
+    "5",
+	"oreEndstoneOlivine",
+    "5",
+	"oreEndstoneTopaz",
+    "5",
+	"oreEndstoneTanzanite",
+    "5",
+	"oreEndstoneAmethyst",
+    "5",
+	"oreEndstoneOpal",
+    "5",
+	"oreEndstoneJasper",
+    "5",
+	"oreEndstoneBlueTopaz",
+    "5",
+	"oreEndstoneGarnetRed",
+    "5",
+	"oreEndstoneGarnetYellow",
+    "5",
+	"oreEndstoneJade",
+    "5",
+	"oreEndstoneVinteum",
+    "5",
+	"oreEndstoneLapis",
+    "5",
+	"oreEndstoneLazurite",
+    "5",
+	"oreEndstoneMysteriousCrystal",
+    "5",
+	"oreEndstoneInfusedFire",
+    "5",
+	"oreEndstoneInfusedAir",
+    "5",
+	"oreEndstoneInfusedWater",
+    "5",
+	"oreEndstoneInfusedEarth",
+    "5",
+	"oreEndstoneInfusedOrder",
+    "5",
+	"oreEndstoneInfusedEntropy",
+    "5",
+	"oreEndstoneNetherStar",
+    "5"
+  ],
+  "radius": 32,
+  "cost": 1,
+  "focusModId": "EMT",
+  "focusName": "EMT_GTBLOCK_CASEING",
+  "focusMeta": 8
```

Comment:
your description says 'draconic casing' (meaning Draconium Fusion casing, meta 9) but meta 8 seems to be Bloody Ichorium Fusion Casing

Reply:
ah you also say tier 1. guess this is the intended one then.

Reply:
Ah yeah, that's a valid point. It technically is a bit misleading now that I think about it but is still under that technical classification by the mod. Hopefully not a lot of people get too confused by it